ChangesToTheExistingSpecification.html

```diff
@@ -0,0 +1,498 @@
+<emu-clause id="modifications">
+  <h1>Modifications to existing internal algorithms</h1>
+  <p>Modifications to existing algorithms are <ins>indicated like this</ins>.</p>
+
+  <emu-clause id="ValidateAndApplyPropertyDescriptor">
+    <h1>9.1.6.3 ValidateAndApplyPropertyDescriptor(O, P, extensible, Desc, current)</h1>
+    <p>When the abstract operation ValidateAndApplyPropertyDescriptor is called with Object O, property key P, Boolean value extensible, and property descriptors Desc, and current the following steps are taken:</p>
+
+    <p>This algorithm contains steps that test various fields of the Property Descriptor Desc for specific values. The fields that are tested in this manner need not actually exist in Desc. If a field is absent then its value is considered to be false.</p>
+
+    <emu-note>If undefined is passed as the O argument only validation is performed and no object updates are performed.</emu-note>
+    <emu-alg>
+      1. Assert: If _O_ is not *undefined* then _P_ is a valid property key.
+      1. <ins>Let _changeType_ be a string, initially set to `"reconfigure"`.</ins>
+      1. If _current_ is *undefined*, then
+        1. If _extensible_ is *false*, then return *false*.
+        1. Assert: _extensible_ is *true*.
+        1. If IsGenericDescriptor(_Desc_) or IsDataDescriptor(_Desc_) is *true*, then
+          1. If _O_ is not *undefined*, then create an own data property named _P_ of object _O_ whose [[Value]], [[Writable]], [[Enumerable]] and [[Configurable]] attribute values are described by Desc. If the value of an attribute field of Desc is absent, the attribute of the newly created property is set to its default value.
+        1. Else Desc must be an accessor Property Descriptor,
```

Comment:
This algorithm in general is missing some punctuation; Desc and current need `_`s, and true/false need `*`s